src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -512,6 +512,11 @@ export default {
 	width: 100%;
 	flex-grow: 1;
 
+	&.collapsed  .crumb:last-child {
+		min-width: 100px;
+		flex-shrink: 1;
+	}
```

Comment:
Please use BEM syntax
```suggestion
	&.crumb--collapsed  .crumb:last-child {
		min-width: 100px;
		flex-shrink: 1;
	}
```